string link_name
---
bool status